src/validation/regex/regex.validation.ts

```diff
@@ -0,0 +1 @@
+export const INVALID_CHARACTERS_FULL_NAME = /^[-,.:; 0-9A-Z&@$£¥€'"«»''""?!/\\()[\]{}<>*=#%+ÀÁÂÃÄÅĀĂĄÆǼÇĆĈĊČÞĎÐÈÉÊËĒĔĖĘĚĜĞĠĢĤĦÌÍÎÏĨĪĬĮİĴĶĹĻĽĿŁÑŃŅŇŊÒÓÔÕÖØŌŎŐǾŒŔŖŘŚŜŞŠŢŤŦÙÚÛÜŨŪŬŮŰŲŴẀẂẄỲÝŶŸŹŻŽa-zÀÖØſƒǺǿẀẅỲỳ]*$/;
```

Comment:
If this has been moved somewhere central to be generic and for reuse then suggest changing constant name to remove _FULL_NAME at the end.

Reply:
yes, I missed that, I thought the regex was for valid chars? so should just be VALID_CHARACTERS

Reply:
More or less said the same above, so changed this.